.github/workflows/ci-master.yml

```diff
@@ -7,13 +7,13 @@ on:
 
 jobs:
 
-    # runs just the main & benchmarking tests on py36/37/38 and mac/windows
+    # runs just the main & benchmarking tests on py37/38 and mac/windows
     test:
       runs-on: ${{ matrix.os }}
       strategy:
         matrix:
           os: [macos-latest, windows-latest]
-          python-version: [3.6, 3.7, 3.8]
+          python-version: [3.7, 3.8]
```

Comment:
Why drop 3.6?

Reply:
Because the more recent NumPy (which is required by the more recent Qiskit) requires Python >=3.7.
See e.g. left column of https://pypi.org/project/numpy/

Reply:
Got it, thanks. Should we update setup.py with [python_requires](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires)?